### PR TITLE
Create nginx.md

### PR DESCRIPTION
**Pull request by ankytastic**
## This PR fixes...

_This PR adds the Dictionary definition of "nginx"_

## What I did...

_Add dictionary definition of "nginx"._

## How to test...

_In a list format, tell us how to test this PR._
Ex:

1. Navigate to /Dictionary
2.Search "nginx"
3.See the definition and perspective show up under "nginx"
4.Check grammar and typos

## I learned...

_Learnt it is called as "engine-x"_

### CODE REVIEW

**Comment by ankytastic** [comment]
[@cherryOnTechAdmin ](https://github.com/cherryontech) please accept my pr and label it for hacktoberfest

File: src/assets/content/dictionary/nginx.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: nginx
+definition: nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server, originally written by Igor Sysoev.
```
Comment (alliequintano) [inline src/assets/content/dictionary/nginx.md]:
```suggestion
definition: Nginx [engine x] is an HTTP and reverse proxy server, a mail proxy server, and a generic TCP/UDP proxy server, originally written by Igor Sysoev.
```

**Review by alliequintano — CHANGES_REQUESTED** [review]
Hi @ankytastic, thank you for contributing to our tech dictionary! Your definition looks good, i've just made a tiny capitalization fix. 

For the perspective, what we are really looking for is to consider a role on a software team such as QA, developer, designer, etc., and write a perspective from that role. I find the example in the [CONTRIBUTING.md ](https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md) helpful to clarify this. Also keep in mind how the value will be evaluated in the sentence, which you can see on the site either testing locally or via the Netlify deploy preview. You can find additional instructions in the CONTRIBUTING.md on that as well.

image

File: src/assets/content/dictionary/SPA.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ definition: SPA stands for Single Page Application. It is a very common way of p
 sources:
 - sourceurl: https://betterprogramming.pub/how-to-implement-a-single-page-application-using-react-router-cc6b9e5c3aec#:~:text=SPA%20stands%20for%20Single%20Page,content%20without%20refreshing%20the%20website.
 perspectives:
-- meaning: SPA is used to make the user experience feel a lot smoother. You can tell when it’s a SPA or multi-page application when navigating between menus often because a multi-page application will reload, making the whole UI blink fast depending on the content.
+- meaning: a way for me to make the user experience feel a lot smoother. You can tell when it’s a SPA or multi-page application when navigating between menus often because a multi-page application will reload, making the whole UI blink fast depending on the content
```
Comment (alliequintano) [inline src/assets/content/dictionary/SPA.md:7]: Did you mean to introduce these fixes with this PR?

**Pull request closed without merging**